include/nlohmann/json.hpp

```diff
@@ -1004,13 +1004,10 @@ class basic_json
             // move the top-level items to stack
             if (t == value_t::array)
             {
-                stack.reserve(array->size());
                 std::move(array->begin(), array->end(), std::back_inserter(stack));
             }
             else if (t == value_t::object)
             {
-                stack.reserve(object->size());
```

Comment:
This one is actually okay because it's not in a loop.

Reply:
Done.